config/routes.rb

```diff
@@ -20,6 +20,9 @@
   # Twilio Request URL for inbound SMS
   post '/api/sms/receive' => 'sms#receive'
 
+  # Image upload API for doc auth
+  post '/api/verify/upload' => 'idv/image_upload#upload'
```

Comment:
Sometimes we try to stick to built-in rails method names/actions to be "RESTful"<sup>1</sup> as much as we can, WDYT of renaming this to `#create`?





---

1. I think "RESTful" is kind of a garbage term and can mean anything... in this case I'm just saying let's use the default names in rails for CRUD verbs